active/radiobuttons/RadioButtons_Spec.md

```diff
@@ -0,0 +1,271 @@
+|:--:|:--:|
+| ![alt text](noselecteditem_notabfocus.png) | ![alt text](noselecteditem_tabfocus.png)|
+
+## Keyboard navigation
+When you have a single column list of RadioButton control options and you have already put focus onto an item, you can navigate the RadioButtons list in logical sequential order using the keyboard.
+
+- The up or down arrow keys will move to the "next" or "previous" logical item (defined in markup)
+- The left or right arrow keys will move spatially
+
+![alt text](keyboardnav_singlecol.png)
+
+This means that in purely vertical RadioButton group lists, left/right arrow keys will not do anything, but up/down will still navigate as expected. However, in purely horizontal RadioButton group lists, left and right will navigate the same way as up and down.
+
+>This behavior is defined more below when we discuss multi-column RadioButton group lists.
+
+### Keyboard navigation with multiple columns
+The keyboarding behavior is the same as the single-column navigation, it just wraps to the next column when there is more than one defined.
+
+![alt text](keyboardnav_multicol.png)
+
+## Selection follows focus
+When you are navigating a RadioButtons list via the keyboard, as focus is placed on a previous or next RadioButton item, that item will also get selected/checked. This means the previous item that was selected/checked will be unselected, and the currently focused one will be instead.
+
+|Before keyboard navigation | After keyboard navigation|
+|:--|:--|
+| ![alt text](2selected_nonav.png) | ![alt text](3selected_yesnav.png)|
+| Focus is on the "2" RadioButton, and it is shown as selected | The down or right arrow key has been pressed, so focus was moved to the "3" RadioButton, thus selected 3 and unselected 2. |
+
+### No Selection follows focus with gamepad
+If you're navigating the RadioButtons container with a gamepad controller, the selection follows focus behavior will be disabled. When navigating a RadioButtons control, no gamepad engagement is necessary to initiate navigation or to select RadioButton elements within the group.
+
+## Accessibility behavior
+
+Below is a detailed table of, depending on the user interaction, what the narrator is expected to say.
+
+|Upon *first* putting focus into the group | Navigating to a selected item |
+|:--|:--|
+| "Group name" RadioButton collection, x of N selected | RadioButton "name" selected, x of N |
+|"Group name" RadioButton collection control, none selected| RadioButton "name" not selected, x of N <br> *(If navigating with shift-arrow keys, meaning no selection following focus)* |
+
+## No Wrapping
+The RadioButtons group does not wrap. This is because when using a screen reader, a sense of boundary and clear indication of ends versus beginnings is lost and thus making it difficult for vision accessible users to navigate the list easily. There is also no enumeration with a RadioButtons group, since groups like this are meant to have a resonable number of items within.
```

Comment:
```suggestion
The RadioButtons group does not wrap. This is because when using a screen reader, a sense of boundary and clear indication of ends versus beginnings is lost and thus making it difficult for blind users to navigate the list easily. There is also no enumeration with a RadioButtons group, since groups like this are meant to have a resonable number of items within.
```

I've heard that "blind" is the more appropriate term